static/src/stylesheets/module/reader-revenue/_moment-epic.scss

```diff
@@ -1,5 +1,5 @@
 // TBD: Add all moment epic class names here when they are available
-[class*='contributions__epic--2019-10_contribs_environment-pledge-moment'] {
+[class*='contributions__epic--2019-10-14_moment_climate_pledge'] {
```

Comment:
This is the agreed upon prefix for all epic test names in this Moment

Reply:
👍

Reply:
okay just to call attention to this: i am hesitant to put a day in the date because we might start tests later in the moment